src/Behavioral.Automation.Selenium/Services/DriverService.cs

```diff
@@ -114,10 +115,10 @@ public ReadOnlyCollection<IWebElement> FindElementsByXpath(string path)
         /// <param name="script">Script text</param>
         /// <param name="args">Script arguments</param>
         /// <returns>ReadOnlyCollection of IWebElement objects</returns>
-        public object ExecuteScript(string script, params object[] args)
+        public void ExecuteScript(string script, params object[] args)
```

Comment:
I guess we should return object here, 'cause there are some executions of this method that checks for result of script execution

Reply:
Edited